src/components/graph/Graph.jsx

```diff
@@ -358,11 +358,74 @@ export default class Graph extends React.Component {
         this.pauseSimulation();
     }
 
+    _toggleNodeConnection(targetNodeId, allConnections) {
```

Comment:
`_toggleNodeConnection` and `_getLeafNodeConnections` could be moved into `graph.helper.js`

Reply:
Agree completely, at the time of the submission I decided to wait for our input where would be the best place for these functions :)

Reply:
These have been moved! :)